lib/docker.js

```diff
@@ -51,7 +51,8 @@ function runImage(imageId, cwd, command, mount) {
   var mountDir = crossPlatformCwd(cwd);
   var mountComponent = mount ? `-v "${mountDir}:/app/src"` : '';
   var envArgComponent = directory.getFormattedEnvArgComponent(cwd);
-  var runCommand = `run -w /app/src -p 3000:3000 --rm -it ${mountComponent} ${envArgComponent} ${imageId} sh -c "${command}" || true`;
+  var commandWithPort = command.replace("$PORT", 3000).replace("${PORT}", 3000)
+  var runCommand = `run -w /app/src -p 3000:3000 --rm -it ${mountComponent} ${envArgComponent} ${imageId} sh -c "${commandWithPort}" || true`;
```

Comment:
I made some more improvements to this. Also, I don't think the `/app/src` dir is going to work well for compiled apps. Can we make this variable?